src/js/form-builder.js

```diff
@@ -730,9 +731,43 @@
 
       advFields.push(textAttribute('maxlength', values));
 
-      return advFields.join('');
-    };
-
+      //+ gimigliano
+      if(opts.typeUserAttrs[values.type]) {}

```

Comment:
No are 3 differents blocks  don't see this page but find them in de .js file. It's more easy